Krivorotova Tatyana/reverseString.js

```diff
@@ -0,0 +1,10 @@
+/*   reverseString   */
+function reverseString(str){
+    str = str.split(' ');
```

Comment:
try to not modify arguments of function - in some cases it could lead to unexpected side effects in your apps